modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/tests/OpenAPIProfile.testcase

```diff
@@ -19,6 +19,28 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanLoadAPIExplorerWithProxy {
+		property portal.acceptance = "true";
+		property portal.proxy.path = "portal";
+		property test.liferay.virtual.instance = "false";
```

Comment:
Apache is configed on CI to `ProxyPass "http://localhost:8080"` and `ServerName localhost` so that why we need to set virutal.instance=false.